examples/new_maps.py

```diff
@@ -0,0 +1,289 @@
+import healpy as hp
+import numpy as np
+import scipy.constants as constants
+import os
+from noise_calc import Simons_Observatory_V3_SA_noise,Simons_Observatory_V3_SA_beams
+from optparse import OptionParser
+import pysm
+from pysm.nominal import models
+
+parser = OptionParser()
+parser.add_option('--seed', dest='seed',  default=1200, type=int,
+                  help='Set to define seed, default=1200')
+parser.add_option('--nside', dest='nside', default=256, type=int,
+                  help='Set to define Nside parameter, default=256')
+parser.add_option('--pysm', dest='use_pysm', default=False, action='store_true',
+                  help='Set to use PySM spectral energy distributions, default=False')
+parser.add_option('--smooth', dest='use_smoothing', default=False, action='store_true',
+                  help='Set to include gaussian spectral index for dust, default=False')
+parser.add_option('--beta-dust-gaus', dest='beta_dust_gaus', default=False, action='store_true',
+                  help='Set to include gaussian spectral index for dust, default=False')
+parser.add_option('--beta-sync-gaus', dest='beta_sync_gaus', default=False, action='store_true',
+                  help='Set to include gaussian spectral index for sync, default=False')
+
+(o, args) = parser.parse_args()
+
+nside = o.nside
+seed= o.seed
+np.random.seed(seed)
+
+# Dust
+A_dust_BB=5
+EB_dust=2.  
+alpha_dust_EE=-0.42 
+alpha_dust_BB=-0.2
+nu0_dust=353. 
+temp_dust = 19.6
+if o.beta_dust_gaus:
+    beta_dust =  hp.ud_grade(hp.read_map('map_beta_dust.fits', verbose=False), nside_out=nside)
```

Comment:
Is this the map you want to read (I'm asking because you seem to be doing something different for synchrotron.

Reply:
This was the map I was testing things with, I've now fixed it to include the variation in amplitude following the format in beta_gaussian.py